lib/super_good/solidus_taxjar.rb

```diff
@@ -31,6 +31,7 @@ class << self
       attr_accessor :taxable_address_check
       attr_accessor :taxable_order_check
       attr_accessor :test_mode
+      attr_accessor :logger
```

Comment:
This should be `attr_writer` I think. I guess this technically works but you're defining a `logger` method and then immediately overriding it.